src/Microsoft.TemplateEngine.Utils/Timing.cs

```diff
@@ -3,38 +3,36 @@
 
 using System;
 using System.Diagnostics;
-using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.Extensions.Logging;
 
 namespace Microsoft.TemplateEngine.Utils
 {
     public class Timing : IDisposable
```

Comment:
nit: Consider making this a `struct`. Or `sealed`.

Reply:
made sealed